src/components/about-us/FlowerDecoration.svelte

```diff
@@ -0,0 +1,11 @@
+<script>
+	import FlowerExplosion from '@assets/about-us/flower-explosion.png';
+</script>
+
+<div class="w-full overflow-hidden">
+	<img
```

Comment:
This could become an `.astro` component and use the `<Image />` component for optimizations without much work, right?

Reply:
I converted to .astro and used <Image /> component